app/models/distributions/podcast_distribution.rb

```diff
@@ -1,30 +1,31 @@
 # encoding: utf-8
 require 'prx_access'
 
-module Distributions
-  class PodcastDistribution < ::Distribution
-    include PRXAccess
-    include Rails.application.routes.url_helpers
+class Distributions::PodcastDistribution < Distribution
+  include PRXAccess
+  include Rails.application.routes.url_helpers
 
-    after_commit :add_podcast_to_feeder, on: [:create]
+  def distribute
```

Comment:
Sticking with the activerecord norms ... would this be `distribute!` instead of `distribute`?  Or is that not hip anymore?